test/functional/fixtures/regression/gh-3681/test.js

```diff
@@ -0,0 +1,22 @@
+const expect = require('chai').expect;
+
+describe('[Regression](GH-3681) - Should throw correct errors for non-exising and invisible iframes', function () {
+    it('Invisible iframe', function () {
+        return runTests('./testcafe-fixtures/index.js', 'Invisible iframe', { shouldFail: true, selectorTimeout: 1 })
+            .catch(function (errs) {
+                expect(errs[0]).contains('The element that matches the specified selector is not visible');
+                expect(errs[0]).contains('.switchToIframe(Selector(\'iframe\')');
+            });
+    });
+
+    it('Non-existing iframe', function () {
```

Comment:
We already have [a test](https://github.com/DevExpress/testcafe/blob/master/test/functional/fixtures/api/es-next/iframe-switching/test.js#L56) for this error.
Would you mind just add missing test for invisible iframe error in that file?